README.md

```diff
@@ -15,6 +15,7 @@ This tutorial teaches how to create a [Kitura Swift backend](http://kitura.io) f
 Upon completion of this tutorial there are several [next steps](#next-steps) you can take to add further functionality to the application:
 
 - [Adding a Database to the Kitura FoodServer with Swift-Kuery](AddDatabase.md)
+- [Add a Website Frontend using the Stencil Template Engine](StencilFrontend.md)
```

Comment:
Please change the two references in this file to "Frontend" to be either "Front end" or "Front End" to match the references in the `StencilFrontend.md` file.